frontend-cert/html-projects/chocolate-cookies-recipe/user-stories.md

```diff
@@ -0,0 +1,15 @@
+1. You should have  a`title` element containing the text `Chocolate Chip Cookies Recipe` and a `meta` element with a `charset` attribute set to `UTF-8` in the `head`.
+
+1. You should have an `h1` element with the text `Chocolate Chip Cookies`.
+
+1. You should have a `p` element that introduces the recipe below the `h1`.
+
+1. You should have one `h2` element with the text `Ingredients` for the ingredients section.
+
+1. You should have an unordered list element with at least 4 list items that lists your ingredients below the first `h2` element.
+
+1. You should have a second `h2` element with the text `Method` for the method section.
```

Comment:
```suggestion
1. You should have a second `h2` element with the text `Instructions` for the instructions section.
```

`Method` sounds a little strange to me. We don't have to change it, but I like `Instructions` better.